lib/if.h

```diff
@@ -145,7 +145,7 @@ struct if_stats {
 #define LOSS_PRECISION          0.000003
 #define TE_MEGA_BIT             1000000
 #define TE_BYTE                 8
-#define DEFAULT_BANDWIDTH       10000
+#define DEFAULT_BANDWIDTH       10
```

Comment:
Not in this scope, but would be useful at least to add a comment what this number means (bytes, mbytes, etc.)?

Reply:
good idea. Done